todo-app/db/schema.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  pgTable,
+  serial,
+  varchar,
+  text,
+  time,
+  timestamp,
+  pgEnum,
+} from "drizzle-orm/pg-core";
+export const status = pgEnum("status", ["DONE", "PENDING"]);
+
+export const todoSchema = pgTable("todos", {
+  id: serial("id").primaryKey(),
+  title: varchar("title", { length: 256 }).notNull(),
+  description: text("description").notNull(),
+  time: timestamp("date").defaultNow().notNull(),
+  status: status("status").notNull().default("PENDING"),
```

Comment:
you might use an postgres enum here, as this column can have only two values. https://orm.drizzle.team/docs/column-types/pg#enum